extensions/chaosmesh/generator/go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/fabric8io/kubernetes-client/extensions/chaosmesh/generator
 
 require (
-	github.com/chaos-mesh/chaos-mesh v1.1.1
+	github.com/chaos-mesh/chaos-mesh/api/v1alpha1 v0.0.0-20210728073817-92f5e71c799f
```

Comment:
Is it possible to add a more stable reference (e.g. v1.2.3)?

https://github.com/chaos-mesh/chaos-mesh/commit/92f5e71c799f

Reply:
Let me check this

Reply:
Let's merge for now, eventually this will have to be updated to v2.0.0+